src/traces/treemap/attributes.js

```diff
@@ -139,29 +139,20 @@ module.exports = {
 
         colors: sunburstAttrs.marker.colors,
 
-        opacitybase: {
-            valType: 'number',
-            editType: 'style',
-            role: 'style',
-            min: 0,
-            max: 1,
-            dflt: 0.5,
-            description: [
-                'Sets the base opacity of the headers based on the depth from the entry.',
-                'This options is not available when having a `colorscale`.'
-            ].join(' ')
-        },
-
-        opacitystep: {
-            valType: 'number',
+        depthfade: {
+            valType: 'enumerated',
+            values: [true, false, 'reversed'],
             editType: 'style',
             role: 'style',
-            min: 0,
-            max: 1,
-            dflt: 0.5,
             description: [
-                'Sets the increment for opacity of the headers based on the depth from the entry.',
-                'This options is not available when having a `colorscale`.'
+                'Fades headers towards the background. Background color with opacity of 0.75',
+                'is used as the destination fading color.',
+                'When `marker.colors` are not set within the trace it is defaulted to *false*;',
+                'otherwise it is defaulted to *true*.',
+                'The *reversed* option could be used to invert the direction of fade',
+                'in which the top elements within hierarchy drawn with fully saturated colors',
+                'while the leaves would be faded towards the background color',
+                'This option is not available when a `colorscale` is present.'
             ].join(' ')
```

Comment:
Here's what I would write as `description`:

```
Determines if the sector colors are faded towards the background from the leaves up to the headers.
This option is unavailable when a colorscale is present, defaults to false when marker.colors is set, 
but otherwise defaults to true.
When set to *reversed*, the fading direction is inverted, 
that is the top elements within hierarchy are drawn with fully saturated colors 
while the leaves are faded towards the background color.
```

Reply:
Thanks.
Done in https://github.com/plotly/plotly.js/pull/4242/commits/3b104b143050a6bbeb0853486dd2ac5a3a82802c.